readme.txt

```diff
@@ -29,11 +29,52 @@ To contribute, report issues, or make feature requests use [Github](https://gith
 3. Go to the M Chart Settings and set Highcharts as the Library
 	- WordPress Admin -> Charts -> Settings
 
+== Highcharts Accessibility module ==
```

Comment:
Add all of this to the docs in the wiki instead of the readme. You can link to that section of the docs in the change log though so it's relatively easy to find from the read me but I try to keep extended documentation out of the readme as it gets unwieldy really fast.